src/physics/cam7/physpkg.F90

```diff
@@ -1756,7 +1771,7 @@ subroutine tphysac (ztodt,   cam_in,  &
 
              ! Since we "added" the reserved liquid back in this routine, we need
              ! to account for it in the energy checker
-             flx_cnd(:ncol) = -1._r8*rliq(:ncol)
+             flx_cnd(:ncol) = -1._r8*rliq(:ncol) + prec_sh(:ncol)
```

Comment:
Just checking that this is a general change and not something that is clubbmf specific.